test-data/unit/check-namedtuple.test

```diff
@@ -185,3 +185,35 @@ class C(B): pass
 B(1).b
 C(2).b
 [builtins fixtures/property.py]
+
+[case testNamedTupleAsDict]
+from collections import namedtuple
+
+X = namedtuple('X', ['x', 'y'])
+x = None  # type: X
+a = x._asdict()
```

Comment:
As with `_replace`, I don't know how to return `OrderedDict`, so it is untested.